docs/runtimes/git-repo-as-hub.ipynb

```diff
@@ -0,0 +1,152 @@
+{
```

Comment:
<p>they <s>must</s> be stored in the same folder as the function.yaml - it not mandatory they can be stored in the same folder  </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/3190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>